elasticsearch/gen-yaml/gen.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit -o pipefail -o nounset
+
+# Fetch from helm repo: https://github.com/bitnami/charts/tree/main/bitnami/elasticsearch
+helm repo add bitnami https://charts.bitnami.com/bitnami
+
+helm template "elasticsearch" bitnami/elasticsearch --version "${BITNAMI_ES_RELEASE}" \
+  --set commonAnnotations."app\.uw\.systems\/repos"="https://github.com/utilitywarehouse/shared-kustomize-bases/tree/main/elasticsearch" \
+  --set auth.existingSecret="elasticsearch" \
+  --set global.kibanaEnabled="true" \
+  --set metrics.enabled="true" \
+  --set master.replicaCount="3" \
+  --set master.masterOnly="false" \
+  --set data.replicaCount="0" \
+  --set coordinating.replicaCount="0" \
+  --set ingest.replicaCount="0" \
+  --set master.resources.requests.cpu="250m" \
+  --set master.resources.limits.cpu="2000m" \
+  --set master.resources.requests.memory="4Gi" \
+  --set master.resources.limits.memory="8Gi" \
+  --set master.persistence.size="100Gi" \
+  --set sysctlImage.enabled="false" \
```

Comment:
Here is configuration for [max_map_count](https://github.com/utilitywarehouse/tf_kube_ignition/blob/master/common.tf#L191)
Amount of file descriptors [can be checked via ES itself](https://www.elastic.co/guide/en/elasticsearch/reference/current/file-descriptors.html)
It's almost double of the minimal requirement: 
```
{
  "nodes" : {
    "aLIIIyZ3SsO545igsJxcbw" : {
      "process" : {
        "max_file_descriptors" : 1048576
      }
    },
    "ujRnujmnTTKpvgJ9hLr7ZQ" : {
      "process" : {
        "max_file_descriptors" : 1048576
      }
    },
    "5y8NYeZvS2CsqOexZl8ofg" : {
      "process" : {
        "max_file_descriptors" : 1048576
      }
    }
  }
}
```

There are multiple ES running in UW, so those parameters must already have the values satisfying this software. Otherwise, while testing the manifest, I would encounter errors like 
`max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]`

Reply:
Indeed, I was expecting that, just please document this.

Reply:
In next PR I will move the parameters to values.yaml, and I will document it there. It would be awkward to do in multiline comment